frontend/src/pages/Home/Pokemon.tsx

```diff
@@ -0,0 +1,77 @@
+import React, { useEffect, useState } from 'react';
+import { FormattedMessage } from 'react-intl';
+import { RouteComponentProps } from 'react-router';
+import { makeGetRequest } from 'services/networking/request';
+import Style from './Pokemon.style';
+
+interface Props extends RouteComponentProps<{ id: string }> {}
+interface State {
+  loading: boolean;
+  pokemon: {
+    id: number;
+    name: string;
+    height: string;
+    weight: string;
+  };
+  error: string;
+}
+
+const Pokemon = (props: Props) => {
+  const { id } = props.match.params;
+  const [loading, setLoading] = useState<State['loading']>(false);
+  const [pokemon, setPokemon] = useState<State['pokemon']>({} as State['pokemon']);
+  const [error, setError] = useState<State['error']>('');
+  useEffect(() => {
+    async function fetchPokemon() {
+      setLoading(true);
+      let fetchedPokemon = {} as State['pokemon'];
+      try {
+        const response = await makeGetRequest(`/pokemon/${id}`);
+        fetchedPokemon = response.body;
+      } catch (caughtError) {
+        setError(caughtError.toString());
+        setLoading(false);
+      }
+      setPokemon(fetchedPokemon);
+      setLoading(false);
+    }
+    fetchPokemon();
+  }, [id]);
+  const baseSpriteUrl = 'https://raw.githubusercontent.com/PokeAPI/sprites/master/sprites/pokemon/';
+  const urls = ['', 'back/', 'shiny/', 'back/shiny/'].map(str => `${baseSpriteUrl + str + id}.png`);
+  const { height, weight } = pokemon;
+  return (
+    <Style.Container>
+      {loading ? (
+        <Style.Loader src="https://trello-attachments.s3.amazonaws.com/5d0ce56e059f0263f02e0155/5d0ce56e059f0263f02e016f/x/f020b178de9e22691149d450613f8f52/loader.svg" />
+      ) : error ? (
+        <Style.Error>
+          Oh no !<br /> The following error happened:
+          <br /> {error}
+        </Style.Error>
+      ) : (
+        <Style.Info>
+          <Style.Title>{pokemon.name}</Style.Title>
+          <Style.Grid>
+            {urls.map((url, i) => (
+              <Style.Image src={url} key={i} />
```

Comment:
Even if it has no impact here, passing index as key props is not a good pattern
https://medium.com/@vraa/why-using-an-index-as-key-in-react-is-probably-a-bad-idea-7543de68b17c